hueplusplus/include/IHttpHandler.h

```diff
@@ -48,7 +50,18 @@ class IHttpHandler
     //! \param adr String that contains an ip or hostname in dotted decimal notation like "192.168.2.1"
     //! \param port Optional integer that specifies the port to which the request should be sent. Default is 80
     //! \return String containing the body of the response of the host
-    virtual std::string sendGetHTTPBody(const std::string &msg, const std::string &adr, int port = 80) const = 0;
+    virtual std::string sendGetHTTPBody(const std::string &msg, const std::string &adr, int port = 80) const
```

Comment:
IHttpHandler should either only contain pure virtual functions, or not be declared as an interface (name BaseHttpHandler or so, maybe as an intermediate base class). That way a different protocol could be supported in the future

Reply:
Changed in 88d61e3